### PR TITLE
Get rid of  random inference in SegNext

### LINKED ISSUE #2237: Exported segnext_s model shows different accuracy per execution
**Describe the bug**
Exported segnext_s model shows different evaluation results per execution
image

**Steps to Reproduce**

1. otx build otx/algorithms/segmentation/configs/ham_segnext_s/template_experimental.yaml --train-data-roots ./tests/assets/common_semantic_segmentation_dataset/train/ --val-data-roots ./tests/assets/common_semantic_segmentation_dataset/val/ --test-data-roots ./tests/assets/common_semantic_segmentation_dataset/val/
2. cd otx-workspace-SEGMENTATION/
3. otx train params --learning_parameters.num_iters 5
4. otx export
5. Try N times otx eval --load-weights ./outputs/20230613_151805_export/openvino/openvino.xml
6. You will see some difference in evaluation results

**Environment:**

- OS: 
- Framework version: 
- Python version:
- OpenVINO version:
- CUDA/cuDNN version:
- GPU model and memory:

### PR DESCRIPTION
**Pull request by kprokofi**
### Summary
resolves #2237

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,11 @@
 
 [![Codecov](https://codecov.io/gh/openvinotoolkit/training_extensions/branch/develop/graph/badge.svg?token=9HVFNMPFGD)](https://codecov.io/gh/openvinotoolkit/training_extensions)
 [![Pre-Merge Test](https://github.com/openvinotoolkit/training_extensions/actions/workflows/pre_merge.yml/badge.svg)](https://github.com/openvinotoolkit/training_extensions/actions/workflows/pre_merge.yml)
+
```
Comment (sovrasov) [inline README.md]: Are these changes necessary? develop is already green

**Review by sovrasov — COMMENTED** [review]
Should e2e for SegNext be re-enabled?

**Comment by kprokofi** [comment]
image
Tests are passed for SegNext-s now

**Comment by kprokofi** [comment]
@jaegukhyun I checked OV inference. It is stable now
I also checked perfomance on Kvasir 24 images. Seems to have no difference with previous version

**Review by sovrasov — APPROVED** [review]

**Review by jaegukhyun — APPROVED** [review]
I tested it locally and the issue can't be reproduced. Thank you so much for this quick fix.
